Update the wordpress example (#44)
ovh/public-cloud-examples: use-cases/wordpress-k8s-mysql/README.md

### Layer 1 : kube

* Create a private network with a [gateway](https://www.ovhcloud.com/en-gb/public-cloud/gateway/)
* Deployment of a managed Kubernetes cluster with its node-pool deployed on the private network

> ### Layer 2 : db & wordpress

Reviewer: ```suggestion
### Layer 2 : MySQL & Wordpress
```

### Create the Terraform variables file (configuration)

> Create the 2 Terraform variables configuration files and fill the needed information.

Reviewer: ```suggestion
Two Terraform variables configuration files have been already created. Edit them to modify the needed information.
```

```bash
vi variables_01.tfvars
```

> Add the following for the kubernetes cluster & customize if needed

Reviewer: ```suggestion
The following variable allows to deploy the Kubernetes cluster in GRA11 region:
```

```
kubernetes = {
    region = "GRA11"
}
```

> 

Reviewer: ```suggestion

If you want to deploy it in another region, edit the file with the wanted region.
```

Add the following & customize if needed
```bash
vim variables_02.tfvars
```

> Add the following for the kubernetes cluster & customize if needed

Reviewer: ```suggestion
The following variable allows to deploy the MySQL DB v8 in GRA region with essential plan and db1-7 flavor:
```

```
database = {
    region       = "GRA"
    plan         = "essential"
    flavor       = "db1-7"
    version      = "8"
} 
```

> 

Reviewer: ```suggestion

Customize the values if needed.

```

### Validate the configuration - 01-kube

```bash
cd 01-kube
terraform init
terraform plan -var-file=../variables_01.tfvars
```

### Create the cluster and the nodes-pool - 01-kube

```bash
terraform apply -var-file=../variables_01.tfvars -auto-approve
```

### Validate the configuration - 02-db-wordpress

```bash
cd ../02-db-wordpress
terraform init
terraform plan -var-file=../variables_02.tfvars
```

### Create the DB, website - 02-db-wordpress

```bash
terraform apply -var-file=../variables.tfvars -auto-approve
```

### Login into Wordpress 

Get the Load Balancer service IP
```bash

> kubectl --kubeconfig=./kubeconfig_file get svc

Reviewer: ```suggestion
kubectl --kubeconfig=./kubeconfig.yml get svc
```

```

The wordpress site is available at this IP. 

The back office is available under `/wp-admin/`. The default user is `user`, the generated password can be retrieved using :
```bash

> kubectl --kubeconfig=./kubeconfig_file get secret --namespace default wordpress -o jsonpath="{.data.wordpress-password}" | base64 -d

Reviewer: ```suggestion
kubectl --kubeconfig=./kubeconfig.yml get secret --namespace default wordpress -o jsonpath="{.data.wordpress-password}" | base64 -d
```

```

> ### Delete the DB and the cluster

Reviewer: ```suggestion
### Delete the DB and the Kubernetes cluster
```
